dbms/src/Storages/DeltaMerge/Filter/In.h

```diff
@@ -57,6 +57,17 @@ class In : public RSOperator
             res = res || rsindex.minmax->checkEqual(pack_id, values[i], rsindex.type);
         return res;
     }
+
+    RSResult roughCheckWithLog(size_t pack_id, const RSCheckParam & param, LoggerPtr log) override
```

Comment:
I think we can just add the "logging" logical to `roughCheck` and remove this method.

Reply:
I don't want to change the origin logical, so I don't add log param for roughCheck.
This debug  method will be removed later, all the reservations for now are because I'm still doing profiling.